index.html

```diff
@@ -214,18 +214,18 @@ <h2>Editors' picks</h2>
             <textarea id="comment" name="comment" cols="50" rows="5" maxlength="250" placeholder="Leave your comment here..." required></textarea>
           </div>
           <div>
-            <label for="opinion">What is your overall opinion about this article?</label>
+            <label>What is your overall opinion about this article?</label>
             <div>
               <label>
-                <input type="radio" name="opinion" id="opinion" value="positive">
+                <input type="radio" name="opinion" value="positive">
                 Positive
               </label>
               <label>
-                <input type="radio" name="opinion" id="opinion" value="negative">
+                <input type="radio" name="opinion" value="negative">
                 Negative
               </label>
               <label>
-                <input type="radio" name="opinion" id="opinion" value="neutral">
+                <input type="radio" name="opinion" value="neutral">
```

Comment:
Your code looks good.
Help me to understand why there are more than one input for radio buttons.